src/PureCloud.Client/Contracts/IRoutingApi.cs

```diff
@@ -1,26 +1,531 @@
 using PureCloud.Client.Models;
+using PureCloud.Client.Models.Queues;
+using PureCloud.Client.Models.Users;
 
 namespace PureCloud.Client.Contracts;
 
 public interface IRoutingApi
 {
-    /// <summary>
-    /// Delete single benefit assessment.
-    /// </summary>
-    Task DeleteRoutingAssessmentAsync(string assessmentId, CancellationToken cancellationToken = default);
-
-    /// <summary>
-    /// Retrieve a single benefit assessment.
-    /// </summary>
+    /// <inheritdoc />
+    Task<AnalyticsQueryResponse> CreateAnalyticsQueuesObservationsQueryAsync(QueueObservationQuery body, CancellationToken cancellationToken = default);
+
+    /// <inheritdoc />
```

Comment:
Interfaces should have full description of each method and each parameter.

The inheritdoc should only be use for implementations but the interface